### DISCUSSION TITLE
Tensorboard disappears after a few epochs...

### DISCUSSION THREAD

**Discussion by F-V-Younesi** [voicepaw/so-vits-svc-fork#865] [Q&A]
After a few epochs( for example 40 epochs) , Tensorboard disappears.... why? and how do I solve it?
I use Colab and GPU.

**Comment by xwkya** [comment]
Which browser are you using @F-V-Younesi

**Reply by F-V-Younesi** [reply]
Chrome
Version 115.0.5790.171 (Official Build) (32-bit)
